### ISSUE TITLE
Enable Raw Prompting

### ISSUE THREAD

**Issue by user-0a** [a16z-infra/cog-llama-template#9]
Hey, thank you so much for making this! Have had a blast using it so far.

Would it be possible to add another parameter "disable_system_prompt", which causes the prompt to be passed in raw i.e. something like this:

```
# skip applying PROMPT_TEMPLATE when disable_system_prompt is True
if not disable_system_prompt:
  prompt = PROMPT_TEMPLATE.format(system_prompt=system_prompt.strip(), instruction=prompt.strip())
```

I would make this a PR but I'm not sure how I would test it and would hate to make a buggy PR haha.

**Comment by DemagnetizedMoralCompass** [comment]
Yeah, this would be very useful when experimenting with non-chat-tuned models. It's not clear to me at the moment if the base models are still given the full instruct style prompt, but they probably shouldn't be since they are not fine tuned for it.

**Comment by niron1** [comment]
@user-0a  @DemagnetizedMoralCompass 
if you need llama-2-7b-chat, my public repo does not alter the prompt: https://replicate.com/niron1/llama-2-7b-chat

**Comment by nateraw** [comment]
I believe this should be resolved now, as we've added `prompt_template` in #78 / #80.

You should be able to set `prompt_template` to `{prompt}` (with the brackets, as its a string formatter), then pass your raw prompt to the `prompt` parameter.
